kvrocks.conf

```diff
@@ -647,12 +647,12 @@ rocksdb.wal_size_limit_mb 16384
 # actual size of the unit read from disk may be smaller if
 # compression is enabled.
 #
-# Default: 4KB
+# Default: 16KB
```

Comment:
4096 is OK here: https://github.com/apache/kvrocks/blob/unstable/src/config/config.cc#L165